.github/workflows/buildAndDocumentation.yml

```diff
@@ -25,31 +23,39 @@ jobs:
 
     steps:
      - uses: actions/checkout@v2
+       with:
+          fetch-depth: 10
+
 
      - name: Install linux deps
        if: matrix.os == 'ubuntu-latest'
        run: |
            sudo apt-get update
-           sudo apt-get install libqglviewer-dev-qt5 libboost-dev libeigen3-dev ninja-build libhdf5-serial-dev libboost-dev libcairo2-dev libgmp-dev libgraphicsmagick++1-dev libfftw3-dev
+           sudo apt-get install zsh libqglviewer-dev-qt5 libboost-dev libeigen3-dev ninja-build libhdf5-serial-dev libboost-dev libcairo2-dev libgmp-dev libgraphicsmagick++1-dev libfftw3-dev
 
      - name: Install macOS deps
        if: matrix.os == 'macOS-latest'
        run: brew install boost ninja eigen gmp
 
+     - name: Get white list of tests
+       run: |
+           echo "::set-output name=WHITE_LIST::$(${{runner.workspace}}/DGtal/.github/workflows/create_whitelist.sh)"
+       id: whitelist
+
      - name: Create Build Environment
        run: cmake -E make_directory ${{runner.workspace}}/build
 
      - name: Configure CMake (linux)
        if: matrix.os == 'ubuntu-latest'
        shell: bash
        working-directory: ${{runner.workspace}}/build
-       run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $CONFIG_GLOBAL $CONFIG_LINUX -G Ninja
+       run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $CONFIG_GLOBAL $CONFIG_LINUX -DDGTAL_RANDOMIZED_TESTING_WHITELIST="${{ steps.whitelist.outputs.WHITE_LIST }}" -G Ninja
```

Comment:
I would prefer `WHITELIST` rather than `WHITE_LIST`

Reply:
Ok I change it, thanks